src/client/auth-client.ts

```diff
@@ -496,6 +505,61 @@ export class AuthClient {
     checkInterruption();
   }
 
+  public async loginWithIcrc29(options?: AuthClientLoginIcrc29Options): Promise<void> {
+    // Merge the passed options with the options set during creation
+    const loginOptions = mergeLoginOptions(this._createOptions?.loginOptions, options);
+
+    // Set default maxTimeToLive to 8 hours
+    const maxTimeToLive = loginOptions?.maxTimeToLive ?? DEFAULT_MAX_TIME_TO_LIVE;
+
+    // Create the URL of the IDP. (e.g. https://XXXX/#authorize)
```

Comment:
Should this not be 
```suggestion
    // Create the URL of the IDP. (e.g. https://XXXX/authorize)
```

Reply:
Good catch! Even more, it should also include the `openid` query param. I'll add it.